README.md

```diff
@@ -181,6 +181,30 @@ Client.restore(id, :recursive => true)
 client.restore(:recursive => true)
 ```
 
+If you want to validate uniqueness but want to include deleted records:
+
+``` ruby
+class Client < ActiveRecord::Base
+  acts_as_paranoid
+
+  validates :name, uniqueness: { paranoia: :with_deleted }
```

Comment:
Don't custom paranoid gem.You can write:
validates :name, uniqueness: {conditions: ->{with_deleted}}